_schemaV2.graphql

```diff
@@ -8874,6 +8876,7 @@ type Profile {
   isPubliclyVisible: Boolean
   isPublished: Boolean
   name: String
+  profileArtistsLayout: String
```

Comment:
Are these schema changes intentional?

Reply:
I suspect they're not

Reply:
Umm, no 🤔 
I wonder why that changed. Good catch, @starsirius!

Reply:
I just reverted the file but they got automatically changed again on the pre-commit hook

Reply:
Hmmm, maybe you need to rebase?

Reply:
Super messy git state 🤦 
Should be fine now. Sorry, folks.